region/wreckedship/main/Bowling Alley.json

```diff
@@ -893,6 +893,29 @@
       "flashSuitChecked": true,
       "note": ["This requires a first frame Spikesuit unmorph."]
     },
+    {
+      "link": [2, 2],
```

Comment:
- I think this one you put a insane+ tech on it.
- I needed a second reserve to reliably get out of the spikes (second spike hit)
- I'd like to see a version that ends at statue.  Getting out of spikes is easier over there too.